Websites/perf.webkit.org/public/v3/models/commit-log.js

```diff
@@ -42,7 +42,7 @@ class CommitLog extends DataModelObject {
     revision() { return this._rawData['revision']; }
```

Comment:
Wonder if this function should return an r-prefixed string if the revision is an integer. Alternatively, could do that in `urlForRevision` and `urlForRevisionRange`

Reply:
For that purpose, we already have `label()`. And changing the behavior of this function may result in a huge refactor as it's used widely.

Reply:
I can see why changing `revision` would be a problem, not `urlForRevision`, though.

Reply:
I started with this approach. But in order to do that, `urlForRevision` will need to explicitly call `CommitLog. _repositoryType` or replicate the code. The former approach coupling two modules together. The latter approach is not DRY.
So I ended up with using identifier whenever it's available.

Reply:
Can we not just check if the string is an integer? It's not perfect, but if you have a full hash, the likelihood a full hash being entirely numbers is basically impossible.

Reply:
This logic is too much custom and we don't guarantee revision is always integer, it can also be digit only string.

Reply:
Right, that's my point. If you're talking to commits.webkit.org and you get an integer or string that is only digits, that's definitely a revision.